### ISSUE TITLE
I accidently introduced a nonexistant   :term: `bool` into doc.

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbem#254]
Used the invalid term in cim_obj.py CIMProperty.tomof().  Should have been (class: `py:bool`)

Propose fixing this as part of fix for issue #223

**Comment by andy-maier** [comment]
I also ran into it and have fixed it as part of PR #256, which does some more cleanup of the Sphinx markup.

**Comment by andy-maier** [comment]
The original issue was fixed by PRs #260 (Karl) and #256 (Andy) in the same way.

However, PR #260 in addition added "Default: xxx" to the properties, repeating the defaults defined for the arguments. During merge conflict resolution, I did not realize that and dropped them.

**Discussion point:** So far, we are not repeating argument defaults defined in the Python function, in the docstring. They are shown in the documentation in the function header.

Need to discuss which way to go there. My proposal is not to repeat them.
